core/src/bench/scala/ai/h2o/sparkling/bench/DataFrameConverterBenchSuite.scala

```diff
@@ -140,4 +141,10 @@ class DataFrameConverterBenchSuite extends BenchSuite with SharedH2OTestContext
     val hf = hc.asH2OFrame(df)
     hf.remove()
   }
+
+  private def sparseVector(len: Int, elements: Int, rng: Random = Random): org.apache.spark.ml.linalg.SparseVector = {
+    assert(elements < len)
+    val data = (1 to elements).map(_ => rng.nextInt(len)).sortBy(identity).distinct.map(it => (it, rng.nextDouble()))
+    Vectors.sparse(len, data).toSparse
```

Comment:
nit: Isn't `.toSparse` extra here?

Reply:
Just changed, interestingly not :( The method returns SparseVector, but in the signature it specifies just `Vector`